src/LtiAssignmentsGradesService.php

```diff
@@ -116,6 +116,18 @@ public function getLineItems(): array
         return $lineitems;
     }
 
+    public function getLineItem(string $url): array
```

Comment:
I liked what you had originally where it returned a `LtiLineItem` object instead of an array. Most of the code that uses it will probably have to instantiate that LtiLineItem anyways, so might as well do it for them.